content/cloud-docs/api-docs/private-registry/providers.mdx

```diff
@@ -282,8 +356,55 @@ curl \
   --header "Content-Type: application/vnd.api+json" \
   https://app.terraform.io/api/v2/organizations/my-organization/registry-providers/public/hashicorp/aws
 ```
+### Sample Response (Private Provider)
+
+```json
+{
+  "data": {
+    "id": "prov-cmEmLstBfjNNA9F3",
+    "type": "registry-providers",
+    "attributes": {
+      "name": "aws",
+      "namespace": "hashicorp",
+      "version-statuses": [
+        {
+          "version": "3.1.1"
+        }
+      ],
```

Comment:
Could we please remove the `version-statuses` attribute from this response?

Reply:
done!